src/main/java/com/shootdoori/match/joinWaiting/controller/JoinWaitingController.java

```diff
@@ -0,0 +1,105 @@
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class JoinWaitingController {
+
+    private final JoinWaitingQueryService joinWaitingQueryService;
+    private final JoinWaitingCommandService joinWaitingCommandService;
+
+    public JoinWaitingController(JoinWaitingQueryService joinWaitingQueryService,
+        JoinWaitingCommandService joinWaitingCommandService) {
+        this.joinWaitingQueryService = joinWaitingQueryService;
+        this.joinWaitingCommandService = joinWaitingCommandService;
+    }
+
+    @PostMapping("/api/teams/{teamId}/join-waiting")
+    public ResponseEntity<JoinWaitingResponseDto> create(
+        @PathVariable Long teamId,
+        @LoginUser Long loginUserId,
+        @RequestBody JoinWaitingRequestDto requestDto
+    ) {
+        return new ResponseEntity<>(
+            joinWaitingCommandService.create(teamId, 1L, requestDto), HttpStatus.CREATED);
+    }
+
+    @PostMapping("/api/teams/{teamId}/join-waiting/{joinWaitingId}/approve")
+    public ResponseEntity<JoinWaitingResponseDto> approve(
+        @PathVariable Long teamId,
+        @PathVariable Long joinWaitingId,
+        @LoginUser Long loginUserId,
+        @RequestBody JoinWaitingApproveRequestDto requestDto
+    ) {
+        return new ResponseEntity<>(
+            joinWaitingCommandService.approve(teamId, joinWaitingId, loginUserId, requestDto),
+            HttpStatus.OK);
+    }
+
+    @PostMapping("/api/teams/{teamId}/join-waiting/{joinWaitingId}/reject")
+    public ResponseEntity<JoinWaitingResponseDto> reject(
+        @PathVariable Long teamId,
+        @PathVariable Long joinWaitingId,
+        @LoginUser Long loginUserId,
+        @RequestBody JoinWaitingRejectRequestDto requestDto
+    ) {
+        return new ResponseEntity<>(
+            joinWaitingCommandService.reject(teamId, joinWaitingId, loginUserId, requestDto),
+            HttpStatus.OK);
+    }
+
+    @PostMapping("/api/teams/{teamId}/join-waiting/{joinWaitingId}/cancel")
+    public ResponseEntity<JoinWaitingResponseDto> cancel(
+        @PathVariable Long teamId,
+        @PathVariable Long joinWaitingId,
+        @LoginUser Long loginUserId,
+        @RequestBody JoinWaitingCancelRequestDto requestDto
+    ) {
+        return new ResponseEntity<>(
+            joinWaitingCommandService.cancel(teamId, joinWaitingId, loginUserId, requestDto),
+            HttpStatus.OK);
+    }
+
+    @GetMapping("/api/teams/{teamId}/join-waiting")
+    public ResponseEntity<Page<JoinWaitingResponseDto>> findPending(
+        @PathVariable Long teamId,
+        @RequestParam(defaultValue = "PENDING") JoinWaitingStatus status,
+        @RequestParam(defaultValue = "MEMBER") JoinWaitingType type,
+        @PageableDefault(size = 10, sort = "timeStamp.createdAt", direction = Sort.Direction.DESC) Pageable pageable
```

Comment:
```suggestion
        @PageableDefault(sort = "timeStamp.createdAt", direction = Sort.Direction.DESC) Pageable pageable
```
이미 기본사이즈가 10이라 필요 없을 것 같네요!

Reply:
확인했습니다!